Test/ParallelPrimitives/RadixSortKernels.h

```diff
@@ -113,184 +113,6 @@ extern "C" __global__ void CountKernel( int* gSrc, int* gDst, int gN, int gNItem
 
 
 
-extern "C" __global__ void CountKernel_tmp( int* gSrc, int* gDst, int gN, int gNItemsPerWI, const int START_BIT, const int N_WGS_EXECUTED )
```

Comment:
The old and unused kernels are removed since they are no longer compatible to the new interfaces.